__tests__/endpoints.test.js

```diff
@@ -205,6 +205,89 @@ describe("app", () => {
           });
         });
     });
+    describe("QUERIES", () => {
+      test("status 200, reviews can be sorted by any valid column", () => {
+        return request(app)
+          .get("/api/reviews/?sort_by=title")
+          .expect(200)
+          .then(({ body: { reviews } }) => {
+            expect(reviews).toBeSortedBy("title", { descending: true });
+          });
+      });
+      test("status 200, reviews can be sorted by specified order", () => {
+        return request(app)
+          .get("/api/reviews/?sort_by=owner&order=asc")
```

Comment:
Hi Anna, just a small suggestion - I would maybe just have this test to be order by as you have already tested for sort_by in the previous test